rotkehlchen/db/settings.py

```diff
@@ -80,6 +80,7 @@ class DBSettings(NamedTuple):
     include_gas_costs: bool = DEFAULT_INCLUDE_GAS_COSTS
     eth_rpc_endpoint: str = 'http://localhost:8545'
     ksm_rpc_endpoint: str = 'http://localhost:9933'
+    dot_rpc_endpoint: str = 'http://localhost:9933'
```

Comment:
As I said I believe those having the same port could be a problem. What if you want both kusama and polkadot?

Reply:
mh both seem to run on the same port by default. How could we fix this issue?

Reply:
What I would do here is store ports for kusama and polkadot as settings for the user in the database and make and option in the UI to change them. The default value could be this `9933`. But maybe @LefterisJP has a better idea?

Reply:
We have talked about this in our standup @jonasW3F . The default value for the port should be different and there is an option to set it already in the front-end, you should allow the customization of the port there

Reply:
I think it is currently possible to set the ports in the front-end for both endpoints. How do we want to handle it? Do we only want to give one default for either KSM or DOT and leave the other empty?

Reply:
I now leave the `dot_rpc_endpoint` blank so that a user can set it in the frontend manually. There might be better ways.